bin/radical-pilot-create-static-ve

```diff
@@ -191,6 +191,9 @@ then
 
         . "$PREFIX"/bin/activate
 
+        # ensure we have pip
+        python3 -m ensurepip
+
         echo -n "update  setuptools pip wheel"
         pip install --no-cache-dir --upgrade setuptools pip wheel || exit 1
```

Comment:
If `ensurepip` installs pip, are you sure it will be on the path? And will it be available as `pip` and not (only) `pip3`? It seems like it would be best to also update this line.
```suggestion
        "$PREFIX"/bin/pip install --no-cache-dir --upgrade setuptools pip wheel || exit 1
```

Reply:
Yes, `ensurepip` will put it in `bin/` which is on the path in the activated env.